slack_cli_hooks/version.py

```diff
@@ -1,2 +1,5 @@
 """Check the latest version at https://pypi.org/project/slack-cli-hooks/"""
-__version__ = "0.0.0.dev2"
+
+import os
+
+__version__ = os.environ.get("SLACK_CLI_HOOKS_VERSION", "0.0.0.dev0")
```

Comment:
Using an environment variable feels much cleaner! I think the `0.0.0.dev0` default makes sense but could also see `development` being used so that `pip install` command stays the same during development:

```sh
$ pip install /path/to/dist/slack_cli_hooks-development-py2.py3-none-any.whl
```

Reply:
Good point 💯 but with `development` I get the following error
```zsh
setuptools.extern.packaging.version.InvalidVersion: Invalid version: 'development'
```
We could use `0.dev0` but I don't think there is a more readable version

Reply:
Oh interesting, thanks for trying this out! This was just an idea I've seen in a few other projects, albeit in different languages, but I think `0.0.0.dev0` is totally fine.